receiver/k8slogreceiver/design.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+The k8slogreceiver is a receiver that receives logs from Kubernetes pods.
+
+```
+   K8s
+   API ─────┐
+            │
+ Docker     │
+  API ────┐ │
+          │ │
+Cri       │ │
+API ────┐ │ │
+        │ │ │
+        ▼ ▼ ▼
+    ┌──────────────┐         ┌───────────┐
+    │    Source    │────────▶│   Poller  │
+    └──────────────┘         └───────────┘
+          │                       │
+          │                       │ 
+          ▼                       │
+    ┌───────────────┐             │
+    │  Reader       │◀────────────┘
+    └───────────────┘
+```
+
+The `Source` will get information about pods from the Kubernetes API and will get the paths will log files exist from the Docker API and the CRI API (Only containerd is supported now).
```

Comment:
Does this depict the `daemonset-stdout` mode?

Reply:
This depict mainly `daemonset-file`. The `daemonset-stdout` is similar, but the Poller can be simplified as there is one path each container.